packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -24,8 +25,9 @@ import 'inspector_tree_flutter.dart';
 
 class InspectorScreen extends Screen {
   const InspectorScreen()
-      : super(
-          'inspector',
+      : super.conditional(
+          id: 'inspector',
+          conditionalLibrary: flutterLibraryUri,
```

Comment:
funny that we missed that the conditionalLibrary property would work great for the inspector!